packages/service-provider-server/src/cli-service-provider.integration.spec.ts

```diff
@@ -433,7 +433,7 @@ describe('CliServiceProvider [integration]', function() {
       });
 
       it('executes the remove op with and resolves the result', () => {
-        expect(result.ok).to.equal(1);
+        expect(result.acknowledged).to.equal(true);
```

Comment:
This is needed to un-break CI because #474 and #477 landed too close to each other :grimacing: